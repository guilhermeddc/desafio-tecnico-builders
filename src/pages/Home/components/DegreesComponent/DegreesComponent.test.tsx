import {render, screen} from '@testing-library/react';
import {convertFahrenheitToCelsius} from 'shared/utils/convertFahrenheitToCelsius';

import {DegreesComponent} from '.';

describe('DegreesComponent', () => {
  it('should render correctly', () => {
    const {container} = render(<DegreesComponent />);

    expect(container).toMatchSnapshot();
  });

  it('should render with default props to type is equal success', () => {
    render(<DegreesComponent temp={250} />);

    expect(
      screen.getByText(`${convertFahrenheitToCelsius(250)}ËšC`),
    ).toBeInTheDocument();
  });
});
