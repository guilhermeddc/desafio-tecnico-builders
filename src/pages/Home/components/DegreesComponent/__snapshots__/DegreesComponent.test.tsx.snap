// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DegreesComponent should render correctly 1`] = `
<div>
  <div
    class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 css-13i4rnv-MuiGrid-root"
  >
    <div
      class="MuiBox-root css-t81ctj"
    >
      <h2
        class="MuiTypography-root MuiTypography-h2 MuiTypography-alignCenter css-1ofdcil-MuiTypography-root"
      >
        -273.15
        ËšC
      </h2>
    </div>
  </div>
</div>
`;
